src/Config/ConfigBase.cs

```diff
@@ -22,6 +22,11 @@
 using System.Linq;
 using System.Runtime.CompilerServices;
 
+#if NET452
+using System.Web.Configuration;
+using System.Web.Hosting;
+# endif
```

Comment:
I think there may be an extra space between "#" and "endif"

Reply:
Fixed.